c++/src/capnp/build.rs

```diff
@@ -95,36 +111,41 @@ fn main() -> eyre::Result<()> {
         } else {
             &[]
         })
-        .map(|s| source_dir.join(s))
-        .try_for_each(|p| {
-            println!(
-                "cargo:rerun-if-changed={}",
-                p.to_str().ok_or_eyre("non–UTF-8 path")?
-            );
+        .chain(if CAPNPC { CAPNPC_SOURCES } else { &[] })
+        .map(|s| (s, source_dir.join(s)))
+        .for_each(|(s, p)| {
+            println!("cargo:rerun-if-changed={}", s);
+            // This copy is only here in case the symlink fails on windows
```

Comment:
does this risk leaving stale files?  
windows with symlinks off and copy: gets copied once, never again if source updates. unsure if dir this is copying to is one that'd be wiped out in any case where the source changes so it doesn't matter

Reply:
iirc hard links work on windows without special configg

Reply:
cxxbuild tries to delete the folder before making it, so assuming this doesn't also break for some reason, it should never risk leaving stale files: https://github.com/dtolnay/cxx/blob/a7898a27add721eca22184acc2d22dd7022c47c4/gen/build/src/out.rs#L70